### PR TITLE
2017-03-12 LWIMIQ

### PR DESCRIPTION
**Pull request by lgalis**
2017-03-12 LWIMIQ

### CODE REVIEW

**Comment by lgalis** [comment]
@miq-bot add_label wip

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+date: 2017-03-12 21:33:30 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+We continue to make great progress as we move closer to the development complete milestone for our Fine release. This week,
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: The urls in the `Featured` section are not resolving correctly - I believe swapping out the `()` for `[]` will correct the problem.

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+84 in the [manageiq-ui-classic](manageiq-ui-classic PRs merged) repo, 17 in the [manageiq-ui-service](manageiq-ui-service PRs merged), 
+with the rest in the various providers' repos.
+ 
+ 
+## Improved
+
+In this [PR](https://github.com/ManageIQ/manageiq/pull/14239), [Jason](https://github.com/Fryguy) moved dynamic method building into DrbRemoteInvoker,
+allowing it to run in the launched automate ruby process directly and not over DRb, preventing random DRb thread issues.
+
+The event collection for the datawarehouse provider now hasa a new field to event streams to keep track of the ext_management_system that carried the event in,
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: s/hasa/has a/

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+The event collection for the datawarehouse provider now hasa a new field to event streams to keep track of the ext_management_system that carried the event in,  
+as added by [Mooli](https://github.com/moolitayer) [here](https://github.com/ManageIQ/manageiq/pull/12773).
+
+## Fixed
+
+[JoeR](https://github.com/jrafanie) fixed a failure that would occur after starting new puma workers and try to gracefully restart apache.
+In his [PR] (https://github.com/ManageIQ/manageiq/pull/14007), puma workers bound to a port from STARTING_PORT to the maximum worker count 
+port (3000 to 3009 if max  worker count is 10). Configure apache at boot with these ports as balancer members.
+
+[Libor]](https://github.com/lpichler) created a fix in this [PR](https://github.com/ManageIQ/manageiq/pull/14229) to allow regex for a
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: Extra `]` in the Libor url

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+MiqExpression::Field containing numbers in associations.
+
+## New
+
+We now support Rails 5.0.2.] (Support Rails 5.0.2).
+
+Check out [Ladas]() Target and Target Collections abstractions [here](#https://github.com/ManageIQ/manageiq/pull/14249).
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: `Ladas` url is missing the target

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+With this new [PR](https://github.com/ManageIQ/manageiq/pull/14225)  - the Ansible Play Data is collected during a Job run.
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: Possibly add Bills page url here as well?

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+The UI for the ems_physical_pages is now available, added by [Julian](https://github.com/juliancheal) in this [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/196) 
+
+
+## Deleted
+
+[Martin](https://github.com/martinpovolny) uses GenericShowMixin to unify and remove #show functionality from middeware controllers -[PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/590)
+He also started on the work of removing the case logic for setting page title and moving the decision back to the controllers in this first Title Refactoring [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/520)
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: Not sure if the extra `-` was added on purpose. If it was please ignore.

May need a `.` at the end of the sentence.

**Comment by lgalis** [comment]
@syncrou - this is still WIP was not ready for review just yet - adding/changing quite a bit still.

**Comment by lgalis** [comment]
@miq-bot remove_label wip

**Comment by lgalis** [comment]
@syncrou, @JPrause - please review

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
+---
+title: "Last Week in ManageIQ: March-ing ahead with the embedded Ansible integration
```
Comment (cybette) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: closing quote `"` needed for title!

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+A busy week again with a lot of progress towards our embedded Ansible Tower features, along with refactoring of existing code, as we move closer to the 
+development complete milestone for our Fine release. Last week, there were 293 commits in 184 PRs across our combined repositories: 
+ - 73 PRs in the [manageiq](manageiq PRs merged) repo
+ - 84 in the [manageiq-ui-classic](manageiq-ui-classic PRs merged) repo
+ - 17 in the [manageiq-ui-service](manageiq-ui-service PRs merged) repo
```
Comment (cybette) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: note usage of round and square brackets,

either: `[link text](url)`

or: `[link text][link id]`
`[link id]: url`

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+[JoeR](https://github.com/jrafanie) fixed a failure that would occur after starting new puma workers and try to gracefully restart apache.
+In his [PR](https://github.com/ManageIQ/manageiq/pull/14007), Apache is configured to boot the ports puma workers are bound to (from STARTING_PORT to the maximum worker count 
+port 3000 to 3009 if max worker count is 10) as balancer members.
+
+[Libor](https://github.com/lpichler) created a fix in this [PR](https://github.com/ManageIQ/manageiq/pull/14229) to allow regex for a
+MiqExpression::Field containing numbers in associations.
+
+## New
+
+[Rails 5.0.2.](Support Rails 5.0.2) is now supported.
```
Comment (cybette) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: Need url for Rails, either in the round brackets or with link id at end of file (which I couldn't find..)

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+Check out [Ladas]() Target and Target Collections abstractions [here](#https://github.com/ManageIQ/manageiq/pull/14249).
```
Comment (cybette) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: Should that be `[Ladas](https://github.com/Ladas)` and `[here](https://github.com/ManageIQ/manageiq/pull/14249)` (there's an extra `#`)

**Comment by JPrause** [comment]
LGTM! Thanks @lgalis 

@d-m-u and @juliancheal can you also give a quick look before today's merge.

**Comment by d-m-u** [comment]
Looks great. Nice job @lgalis!

**Comment by JPrause** [comment]
@lgalis it appears all edits have been completed and checks have passed. If this is ready, we'll get this merged. Is that okay?

**Comment by juliancheal** [comment]
LGTM 👍 Good job Laura.

**Comment by lgalis** [comment]
@JPrause - sounds good

**Comment by JPrause** [comment]
@cybette looks like we're ready for merge. 👍

File: site/_posts/2017-03-12-marching-ahead-embedded-ansible.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+The UI for the physical infrastructure is now available, added by [Julian](https://github.com/juliancheal) in this [PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/196). 
+
+For the REST API, [Daniel](https://github.com/dkorn) added [REST API](https://github.com/ManageIQ/manageiq/pull/13967) support for Alert Definitions (Miq Alert), while [Jillian](https://github.com/jntullo) 
+added the read API for [configuration scripts](https://github.com/ManageIQ/manageiq/pull/14223) and access to the jobs on a [ServiceAnsiblePlaybook](https://github.com/ManageIQ/manageiq/pull/14248).
+
+
+## Deleted
+
+[Martin](https://github.com/martinpovolny) uses GenericShowMixin to unify and remove #show functionality from middeware controllers -[PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/590)
+
```
Comment (syncrou) [inline site/_posts/2017-03-12-marching-ahead-embedded-ansible.md]: @lgalis - Little late to the party - possibly add a space between the '-' and PR?

**Pull request merged**